corehq/apps/hqwebapp/management/commands/build_requirejs.py

```diff
@@ -176,9 +176,10 @@ def _get_html_files_and_local_js_dirs(self):
         for root, dirs, files in os.walk(prefix):
             for name in files:
                 if name.endswith(".html"):
-                    filename = os.path.join(root, name)
-                    if not re.search(r'/partials/', filename):
-                        html_files.append(filename)
+                    filename = filename = os.path.join(root, name)
```

Comment:
one to many `filename =` not sure what that actually does.

Reply:
whoops - 7cbf25a7d67